target/linux/rockchip/image/armv8.mk

```diff
@@ -56,16 +51,13 @@ define Device/pine64_rock64
   DEVICE_VENDOR := Pine64
   DEVICE_MODEL := Rock64
   SOC := rk3328
-  UBOOT_DEVICE_NAME := rock64-rk3328
```

Comment:
why do you remove `UBOOT_DEVICE_NAME` here?

Reply:
[rockchip: move UBOOT_DEVICE_NAME to default](https://github.com/openwrt/openwrt/commit/afff2feb28baecaaf368002d819fb63973cf778a)

It has been merged into default set.